torch/quantization/QConfig.py

```diff
@@ -63,3 +65,25 @@ def __new__(cls, weight):
 
 default_qat_qconfig = QConfig(activation=default_fake_quant,
                               weight=default_weight_fake_quant)
+
+def get_default_qconfig(backend='fbgemm'):
+    if backend == 'fbgemm':
+        qconfig = QConfig(activation=HistogramObserver.with_args(reduce_range=True),
+                          weight=default_per_channel_weight_observer)
+    else:
+        qconfig = QConfig(activation=HistogramObserver.with_args(reduce_range=False),
+                          weight=default_weight_observer)
+    return qconfig
+
+def get_default_qat_qconfig(backend='fbgemm'):
+    if backend == 'fbgemm':
+        qconfig = QConfig(activation=FakeQuantize.with_args(observer=default_l2_observer(reduce_range = True),
```

Comment:
I think this will create the same instance of observer for each FakeQuant - you probably need double with_args, but it gets ugly...